Sources/PayPalWebPayments/PayPalWebCheckoutClient.swift

```diff
@@ -100,16 +100,15 @@ public class PayPalWebCheckoutClient: NSObject {
     /// Starts a web session for vaulting PayPal Payment Method
     /// After setupToken successfullly attaches a payment method, you will need to create a payment token with the setup token
     /// - Parameters:
-    ///   - url: URL created from string value from setupToken API
-    /// - Returns: PayPalVaultResult
-    /// - Throws: PayPalSDK error if vaulting could not complete successfully
-    public func vault(url: URL) {
-        if let setupToken = getQueryStringParameter(url: url.absoluteString, param: "approval_session_id") {
+    ///   - vaultRequest: Request created with url for vault approval and setupTokenID
+    public func vault(_ vaultRequest: PayPalVaultRequest) {
+        if let setupToken = getQueryStringParameter(url: vaultRequest.url.absoluteString, param: "approval_session_id") {
```

Comment:
Should we use the setupToken from the `vaultRequest` instead?

Reply:
Good point, I just resolved the merge conflict and didn't think about it. That would be much easier,

Reply:
addressed here: 981bfc192e1bdc1fb7afc7b1e51719d94a5b9f00
and b597eb7df7e64e90c46812cabb145a9bf358a42b